projects/com.oracle.truffle.llvm.parser.base/src/com/oracle/truffle/llvm/parser/base/facade/NodeFactoryFacade.java

```diff
@@ -77,6 +77,14 @@
     // also update {@link NodeFactoryFacadeComposite} when changing or removing this name
     void setUpFacade(LLVMParserRuntime runtime);
 
+    /**
+     * Get Parser runtime (a temporary required method until we use the new Bitcode Type exclusive).
+     *
+     * @return runtime
+     */
+    @Deprecated
```

Comment:
This is a temporary methode only required by ```findOrAllocateGlobal``` of the old Visitor, so I set it to deprecated to warn when someone try to use this function at other places.